index.html

```diff
@@ -1,99 +1,157 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css"
-      rel="stylesheet"
-    />
-    <link rel="stylesheet" type="text/css" href="styles.css" />
-    <!-- :::FAVICON -->
-    <link rel="shortcut icon" type="image/x-icon" href="favicon/favicon.ico"> <!-- FOR IE -->
-    <link rel="icon" type="image/x-icon" href="favicon/favicon.ico"> <!-- FOR Browser Chrome, Opera, etc -->
-    <link rel="apple-touch-icon" sizes="180x180" href="favicon/apple-touch-icon.png"> <!-- FOR iPhone -->
-    <link rel="icon" type="image/png" sizes="32x32" href="favicon/favicon-32x32.png">
-    <link rel="icon" type="image/png" sizes="16x16" href="favicon/favicon-16x16.png">
-    <link rel="manifest" href="favicon/site.webmanifest">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link
+          href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css"
+          rel="stylesheet"/>
+          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+        <link rel="stylesheet" type="text/css" href="styles.css" />
+        <!-- :::FAVICON -->
+        <link rel="shortcut icon" type="image/x-icon" href="favicon/favicon.ico"> <!-- FOR IE -->
+        <link rel="icon" type="image/x-icon" href="favicon/favicon.ico"> <!-- FOR Browser Chrome, Opera, etc -->
+        <link rel="apple-touch-icon" sizes="180x180" href="favicon/apple-touch-icon.png"> <!-- FOR iPhone -->
+        <link rel="icon" type="image/png" sizes="32x32" href="favicon/favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="favicon/favicon-16x16.png">
+        <link rel="manifest" href="favicon/site.webmanifest">
+<body>
 
-    <!-- :::TITLE -->
```

Comment:
im not sure why this is missing in your code i see you have pulled the latest just over an hour ago but the title needs to be there